pretixdroid/app/src/main/java/eu/pretix/pretixdroid/AppConfig.java

```diff
@@ -49,11 +50,19 @@ public boolean getAutofocus() {
         return prefs.getBoolean(PREFS_KEY_AUTOFOCUS, true);
     }
 
+    public boolean getSound() {
```

Comment:
I think "getSoundEnabled" would be a better name, what do you think? (Same for setSoundEnabled)